python/semantic_kernel/skill_definition/parameter_view.py

```diff
@@ -8,9 +9,27 @@ class ParameterView(SKBaseModel):
     name: str
     description: str
     default_value: str
+    type_: str = "string"
+    required: bool = False
 
-    def __init__(self, name: str, description: str, default_value: str) -> None:
+    def __init__(
+        self,
+        name: str,
+        description: str,
+        default_value: str,
+        type_: str = "string",
+        required: bool = False,
```

Comment:
I'm thinking it makes sense to make required default to true since the standard for most (all?) languages is that params are required unless the language allows for optional or a default.